example/deploy-docs-live-code-contracts.ts

```diff
@@ -0,0 +1,969 @@
+/// How to use:
+///   Ensure the Testfunder account tz1bwsEWCwSEXdRvnJxvegQZKeX5dj6oKEys has at least 2k tokens
+///   for the testnet in use. The script will first check if addresses used in the script are funded,
+///   and top them up if needed. The script will then originate each contract needed for Live Code
+///   examples in Taquito Docs and produce a JSON file with each Contract Identifier and its PKH.
+///   Use the testpad script Docs Live Code Contract Origination (in https://ecadlabs.ontestpad.com/project/18/)
+///   with the desired chain (e.g. hangzhou) to match the originated scripts with their locations in the Docs.
+///   The script will also produce a json file of contracts to use in the code examples in taquito/examples
+///   Execute this script from the ~/taquito/example dir with
+///    node -r ts-node/register ~/taquito/example/deploy-docs-live-code-contracts.ts
```

Comment:
I think the command should be `node -r ts-node/register deploy-docs-live-code-contracts.ts` instead of `node -r ts-node/register ~/taquito/example/deploy-docs-live-code-contracts.ts`. The second one was not working for me.

Reply:
changed thanks